src/game/sound.c

```diff
@@ -329,7 +330,7 @@ bool Sound_Effect(int32_t sfx_num, PHD_3DPOS *pos, uint32_t flags)
     }
 
     int32_t pitch = 100;
-    if (s->flags & SAMPLE_FLAG_PITCH_WIBBLE) {
+    if (g_Config.pitch_sound_effects && s->flags & SAMPLE_FLAG_PITCH_WIBBLE) {
```

Comment:
This may be stupid on my part but I don't trust C with operator precedence, do you think we could wrap `s->flags & …` with braces?

Reply:
Probably.